scripts/experimental/configure_for_workload_identity.sh

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env bash
+
+# Prior to running this script, please ensure that you are running these versions or later:
+# export SPINNAKER_VERSION=release-1.17.x-latest-validated
+# export HALYARD_VERSION=1.26.0
+#
+# This script is intended to be run after the initial setup.sh script completes and Spinnaker is up
+# and running (without Workload Identity enabled).
+#
+# The expected workflow is as follows:
+#   - Generate the properties file by running the setup_properties.sh script
+#   - Modify the properties file to specify the above 2 Spinnaker/Halyard versions (or later versions)
+#   - Run setup.sh
+#   - Once Spinnaker is up and running, run this (configure_for_workload_identity.sh) script
+#
+# Note that this script results in each Spinnaker pod still using the default Kubernetes service account,
+#   and the default service account in the halyard and spinnaker namespaces being bound to one Google
+#   service account (spinnaker-wi-acct). If you want to specify a different Kubernetes service account
+#   for any service, you can do so via the `serviceAccountName` setting described here:
+#   https://www.spinnaker.io/reference/halyard/custom/#kubernetes
+#   You would also need to make the appropriate bindings between that Kubernetes service account and a
+#   Google service account.
+#
+#   The roles assigned are sufficient for deployment to GKE. If you intend to deploy to GCE or GAE, you
+#     will need to assign the appropriate roles to the spinnaker-wi-acct Google service account, similar
+#     to what we do in these helper scripts for the non-Workload Identity setup:
+#       https://github.com/GoogleCloudPlatform/spinnaker-for-gcp/blob/master/scripts/manage/add_gce_account.sh
+#       https://github.com/GoogleCloudPlatform/spinnaker-for-gcp/blob/master/scripts/manage/add_gae_account.sh
+
+bold() {
+  echo ". $(tput bold)" "$*" "$(tput sgr0)";
+}
+
+PROPERTIES_FILE="$HOME/spinnaker-for-gcp/scripts/install/properties"
+
+source "$PROPERTIES_FILE"
+
+bold "Scaling down deployments to 0..."
+kubectl scale deployment spin-clouddriver -n spinnaker --replicas=0
+kubectl scale deployment spin-echo -n spinnaker --replicas=0
+kubectl scale deployment spin-front50 -n spinnaker --replicas=0
+kubectl scale deployment spin-igor -n spinnaker --replicas=0
+kubectl scale deployment spin-kayenta -n spinnaker --replicas=0
+
+bold "Enabling workload identity on cluster $GKE_CLUSTER in project $PROJECT_ID..."
+gcloud beta container clusters update $GKE_CLUSTER \
+  --zone=$ZONE \
+  --identity-namespace=$PROJECT_ID.svc.id.goog \
+  --project=$PROJECT_ID
+
+unset CLUSTER_STATUS
+
+while [ "$CLUSTER_STATUS" != "RUNNING" ]; do
+  CLUSTER_STATUS=$(gcloud beta container clusters describe $GKE_CLUSTER \
```

Comment:
Why are we using the `beta` command here?  (There are a few other places below as well that seem to be using GA functionality but are using the `beta` API surface.)

Reply:
Result of lazy copy/paste. It's not needed for the two `describe` queries, but is needed on the two mutating `update` calls (the workload identity specific flags are only supported via the `beta` API).

Done.